app/src/main/java/org/andstatus/todoagenda/widget/WidgetEntry.java

```diff
@@ -32,6 +35,19 @@ public int getDaysFromToday() {
                 startDate.withTimeAtStartOfDay()).getDays();
     }
 
+    public CharSequence getDaysFromTodayString(Context context, int daysFromToday) {
```

Comment:
Moving the formatting method (which is about presentation) to the data class is incorrect.